backend/internal/service/handler/spotistics/clientcreds.go

```diff
@@ -0,0 +1,51 @@
+package spotistics
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/gofiber/fiber/v2"
+	spotifyauth "github.com/zmb3/spotify/v2/auth"
+
+	"github.com/zmb3/spotify/v2"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+type SpotifyCredentials struct {
+	ClientID     string
+	ClientSecret string
+}
+
+type SpotifyKey struct{}
```

Comment:
```suggestion
type spotifyKey struct{}
```

unexported so can only be used from within this package